packages/sanity/src/core/preview/documentPreviewStore.ts

```diff
@@ -59,6 +65,31 @@ export interface DocumentPreviewStore {
     paths: PreviewPath[],
   ) => Observable<DraftsModelDocument<T>>
 
+  /**
+   * Observes a set of document IDs that matches the given groq-filter. The document ids are returned in ascending order and will update in real-time
+   * Whenever a document appears or disappears from the set, a new array with the updated set of IDs will be pushed to subscribers.
+   * The query is performed once, initially, and thereafter the set of ids are patched based on the `appear` and `disappear`
+   * transitions on the received listener events.
+   * This provides a lightweight way of subscribing to a list of ids for simple cases where you just want to subscribe to a set of documents ids
+   * that matches a particular filter.
+   * @hidden
+   * @beta
+   * @param filter - A groq filter to use for the document set
+   * @param params - Parameters to use with the groq filter
+   * @param options - Options for the observer
+   * @param options - Options for the observer
```

Comment:
Duplicate @param on lines 79 and 80

Reply:
thanks!

Reply:
Fixed by 17ab291a61dc8c79e253ccf562d30b6f80be2a4e